elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -478,6 +525,20 @@ def grade(element_html, data):
         num_initial_correct, true_answer_length = grade_dag(submission, depends_graph, group_belonging)
         first_wrong = -1 if num_initial_correct == len(submission) else num_initial_correct
 
+        if feedback_type == 'first-wrong':
+            for i, block in enumerate(student_answer):
+                if i >= num_initial_correct: break
```

Comment:
```suggestion
            for block in student_answer[:first_wrong + 1]:
```

Reply:
oh wait, this also has problems when first_wrong is -1. I stil think there should be a more elegant solution than the break though...maybe not

Reply:
Yeah, that's the issue was was running into so I opted for this. I'll mull it over and see if I can think of a simpler solution.